lib/std/math/test.zig

```diff
@@ -0,0 +1,177 @@
+const std = @import("../std.zig");
+const print = std.debug.print;
+const meta = std.meta;
+const math = std.math;
+const bitCount = meta.bitCount;
+const nan = math.nan;
+
+// Switch to 'true' to enable debug output.
+var verbose = false;
```

Comment:
The verbose flag should be justified. Why not use comptime log scope here? Tests are relative fast to compile and you also dont obtain the value from the test runner via cmd flag or stuff like that.

Reply:
Yeah, I was thinking this could be hooked up better. Any advice on how to do so? I'm not hugely familiar with running the Zig test suite. What do you mean by comptime log scope? How would I obtain "the value" from the test runner?

Reply:
Use `log.info` like in `heap/logging_allocator.zig`.

Usage from the test runner would be then in the test block or file
`pub const log_level: std.log.Level = .info;`, but this should either be a comment or removed.
Make sure you dont get name clashes, ie prefix log with `std.log`.